examples/arithmetic/tests/test_generated_bindings.rs

```diff
@@ -1,4 +1,5 @@
 uniffi_macros::build_foreign_language_testcases!(
+    "src/arithmetic.idl",
```

Comment:
`build_foreign_language_testcases` now takes an `.idl` file as first argument.

Reply:
I agree, it would be nicer to separate this argument more directly. Can be for a followup if it's a PITA though.